datasets/preprocessed datasets/preprocessed_alhammad.csv

```diff
@@ -0,0 +1,57 @@
+﻿,label,n_words_in_Title,n_words_in_Abstract,text,text_clean,Souza MR,Veado L,Moreira RT,Figueiredo E,Costa H,Silva PC,Mangina E,Sripada SK,de M. Rodrigues E,de S. Carvalho EL,da Silva JP,Reis R,Silva MA,Steinmache I,Mangle AH,Tillmann N,Funk M,Abele S,Hörz T,Schmidt A,Knutas A,Cleary B,Qu W,Zhao Y,Liu B,Colteli O,Grandi X,Ros-Bernal F,Álvarez MC,Guilabert TS,Bernik A,d. Halleux J,Hernández L,Muñoz M,Peña A,Torres C,Gómez-Alvarez MC,Manrique-Losada B,Jimenez DG,Clarisó R,Gan CK,Dubey R,LaToza TD,van der Hoek A,Herbsleb JD,Müller BC,Reise C,Seliger G,Browne K,Anand C,Gosse E,Krauss K,Dodero JM,García-Domínguez A,Faghihi U,Martin D,Brown A,Maldonado-Bouchard S,Uskov V,Sekar B,Berkling K,Kaiser G,Raab F,Lema Moreta L,Palacios MG,Mora A,Arnedo-Moreno J,Matsubara PG,Silva CL,Kheirkhahzadeh AD,Fotaris P,Ibanez MB,Labaj M,Bielikova M,Fuchs M,Fu Y,Dubois DJ,Chirila C,Bartel A,Hagel G,publication_date_2011,publication_date_2014


```

Comment:
not sure if preprocessed dataset should be added to the repository?

Reply:
Also, I would try to avoid directory names with whitespaces, it can get really messy